CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ Please note that this project is released with a [Contributor Code of Conduct](h
 
 `awesome-docker` is a hand-crafted list for high-quality information about Docker and its resources. It should be related or compatible with Docker. If it's just an image built on top of Docker, the project possibly belongs to other [awesome lists](https://github.com/sindresorhus/awesome). You can check the [awesome-selfhosted list](https://github.com/Kickball/awesome-selfhosted) or the [awesome-sysadmin list](https://github.com/n1trux/awesome-sysadmin) as well.
 If it's a **tutorial or a blog post**, they get outdated really quickly so we don't really put them on the list but if it is on a very advanced and/or specific topic, we will consider it!
-If something is awesome, share it (pull request or issue or chat), let us know why and we will help you!
+If something is awesome, share it (pull request or issue or [chat](https://gitter.im/veggiemonk/awesome-docker)), let us know why and we will help you!
```

Comment:
Same idea here:

[pull request](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md#rules-for-pull-request) or [issue](https://github.com/veggiemonk/awesome-docker/issues/new) or [chat](https://gitter.im/veggiemonk/awesome-docker)